src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -300,7 +305,7 @@ const CryptoManagerSettings& CryptoManagerImpl::get_settings() const {
   return *settings_;
 }
 
-bool CryptoManagerImpl::set_certificate(const std::string& cert_data) {
+bool CryptoManagerImpl::SaveCertificateData(const std::string& cert_data) {
   LOG4CXX_AUTO_TRACE(logger_);
```

Comment:
Function doesn't change internal state, it should be const
It would be better function will be called with prefix "Is", IsSaveCertificateData

Reply:
@AKalinich-Luxoft Agree with `const` , disagree with prefix `Is`.

Reply:
@KVGrygoriev also disagree with prefix `Is`
Const added in https://github.com/smartdevicelink/sdl_core/pull/2218/commits/389a64f182249a4154554788f320575a697e2092